test_vm/src/lib.rs

```diff
@@ -763,12 +763,17 @@ impl<'invocation, 'bs> Runtime<&'bs MemoryBlockstore> for InvocationCtx<'invocat
         self.v.get_actor(self.to()).unwrap().balance
     }
 
-    fn resolve_address(&self, addr: &Address) -> Option<Address> {
-        self.v.normalize_address(addr)
+    fn resolve_address(&self, addr: &Address) -> Option<ActorID> {
+        if let Some(normalize_addr) = self.v.normalize_address(addr) {
+            if let &Payload::ID(id) = normalize_addr.payload() {
```

Comment:
Unwrap

Reply:
This maybe also need reserve for some tests

Reply:
Same. We need the outer `if let`, but the inner one can't fail.